src/Sulu/Bundle/AdminBundle/Resources/js/components/Radio/Radio.js

```diff
@@ -1,18 +1,22 @@
 // @flow
 import React from 'react';
 import type {ElementRef, Node} from 'react';
+import classNames from 'classnames';
 import radioStyles from './radio.scss';
 
 type Props = {
+    name: string,
```

Comment:
And if we want to have this property it should also be in the checkbox for consistency.